native-activity/README.md

```diff
@@ -3,19 +3,63 @@
 This is an Android sample that uses [NativeActivity] with `native_app_glue`,
 which enables building NDK apps without having to write any Java code. In
 practice most apps, even games which are predominantly native code, will need to
-call some Java APIs or customize their app's activity further.
+call some Java APIs or customize their app's activity further. It's also worth
```

Comment:
be explicit and say what you mean. (which i think is "so you probably don't want to go this way"?)